sdk/tables/test-resources.json

```diff
@@ -107,6 +114,10 @@
         }
     ],
     "outputs": {
+        "COSMOS_TABLES_ENDPOINT_SUFFIX": {
```

Comment:
There should also be a `STORAGE_TABLES_ENDPOINT_SUFFIX`

Reply:
@christothes I don't see any references to that env var in the tables tests?

Reply:
Oh I see, there's a core variable STORAGE_ENDPOINT_SUFFIX and we currently define that on the backend.